twos-complement/bigint2twos.go

```diff
@@ -0,0 +1,82 @@
+package twoscomplement
+
+import "math/big"
+
+var bigOne = big.NewInt(1)
+
+// ToBytes returns a variable length two's complement byte array representation of the input.
```

Comment:
`// ToBytes returns a byte array of variable length representing the input big.Int in two's complement.`